src/helpers/getWrapContract.ts

```diff
@@ -1,7 +1,8 @@
 import { WarpFactory } from 'warp-contracts'
 import { DeployPlugin } from 'warp-contracts-plugin-deploy'
 
-const warp = WarpFactory.forMainnet().use(new DeployPlugin())
+// const warp = WarpFactory.forMainnet().use(new DeployPlugin())
+const warp = WarpFactory.forTestnet().use(new DeployPlugin())
```

Comment:
Will we merge this into main later?

Reply:
I am using a new deployed contract to testnet so i have to change back the contract address and warp to mainnet before we merge.